includes/class-wc-stripe-webhook-handler.php

```
@@ -1781,6 +1790,74 @@ private function get_payment_method_type_from_charge( $charge ) {
+	private function process_agentic_hook( stdClass $event ) {
+		$event_type = $event->type ?? 'No event type found';
+
+		switch ( $event_type ) {
+			case 'v1.delegated_checkout.customize_checkout':
+				$this->process_agentic_customization_hook( $event );
+				break;
+			case 'v1.delegated_checkout.finalize_checkout':
+				// Coming soon...
+				break;
+			default:
+				WC_Stripe_Logger::error( 'Unsupported agentic hook type: ' . $event_type );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Logger' in method 'process_agentic_hook'. (undefined)

(StaticAccess)